app/models/event.rb

```diff
@@ -0,0 +1,17 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  has_many :attendances, foreign_key: :concerned_event_id
+  has_many :invited_users, through: :attendances
+  validates_presence_of :location, :description, :title, :date
+
+  # def self.past
+  #   self.where("date < ?", Date.today)
+  # end
+  #
+  # def self.upcoming
+  #   self.where("date > ?", Date.today)
+  # end
```

Comment:
- [ ] Remove commented code. If you need to see a previous version you can refer to previous commits.